Create Issue templates (#73)
substrate-developer-hub/substrate-node-template: .github/ISSUE_TEMPLATE/report-a-bug.md

---
name: Report a Bug
about: Report a problem with this template.
title: ""
labels: bug
assignees: ""
---

**Description**

_Tell us what happened. In particular, be specific about any changes you made to this template.
Ideally, provide a link to your project's GitHub repository. Please note that we are not able to

> support any conceivable change to this template project, but the more information you are able to

Reviewer: ```suggestion
support all conceivable changes to this template project, but the more information you are able to
```

provide the more equipped we will be to help._

**Steps to Reproduce**

_Replace the example steps below with actual steps to reproduce the bug you're reporting._

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected vs. Actual Behavior**

_What did you expect to happen after you followed the steps you described in the last section? What
actually happened?_

**Environment**

_Describe the environment in which you encountered this bug. Use the list below as a starting point
and add additional information if you think it's relevant._

- Operating system:

> - Template version/tag:

Reviewer: ```suggestion
- Template version/tag:
- Rust version (run `rustup show`):
```

**Logs, Errors or Screenshots**

> _Please provide the text of any logs or errors that you think may be related to the bug; if

Reviewer: ```suggestion
_Please provide the text of any logs or errors that you experienced; if
```
